src/common/SPELLS_DEATH_KNIGHT.js

```diff
@@ -173,6 +173,26 @@ export default {
     icon: 'spell_shadow_deadofnight',
   },
 
+  //Blood Tier Sets
+  //T20 2 Piece
+  BLOOD_T20_2PC_GRAVEWARDEN: {
```

Comment:
The preferred property naming scheme for set piece definitions is `BLOOD_DEATH_KNIGHT_T20_4SET_BONUS_BUFF`

Reply:
Fixed